mpz-core/src/cointoss/mod.rs

```diff
@@ -0,0 +1,43 @@
+//! A simple 2-party coin-toss protocol.
+//!
+//! # Example
```

Comment:
Does this example serve as a test? How can i run it? Does the CI need to be updated to also run examples?

Reply:
ok, I see. However, I noticed that vscode's code navigation doesnt work for doctests (lmk if it's just me). It creates unnecesary friction when examining the code. Can we duplicate the doctests also as normal tests? 
@sinui0